signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1112,8 +1111,14 @@ async fn sign_bitcoin_transaction() {
         signers.push((ctx, db, kp, network));
     }
 
+    let (broadcast_stacks_tx, _rx) = tokio::sync::broadcast::channel(1);
+
+    let mut stacks_tx_receiver = broadcast_stacks_tx.subscribe();
+    let stacks_tx_receiver_task = tokio::spawn(async move { stacks_tx_receiver.recv().await });
+
     // 3. Check that there are no DKG shares in the database.
```

Comment:
nit: wrong comment?

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/781/commits/2132edc0310007a2d39b6efa0ac302fa6c1e29e6